go-controller/pkg/util/net.go

```diff
@@ -296,3 +296,12 @@ func IPsToNetworkIPs(ips ...*net.IPNet) []*net.IPNet {
 	}
 	return nets
 }
+
+// StringsToIPs takes a slice of strings and returns a slice of net.IPs
+func StringsToIPs(ips []string) []net.IP {
+	s := []net.IP{}
```

Comment:
nit: you can use `s := make([]*net.IPNet, len(ips))` and array indexes just like the method above for that negligible performance gain ;)

Reply:
ack good point. changing this..

Reply:
done.